src/Symfony/Component/Form/Tests/AbstractBootstrap3LayoutTest.php

```diff
@@ -1899,6 +1899,23 @@ public function testMoney()
         );
     }
 
+    public function testMoneyWithoutCurrency()
+    {
+        $form = $this->factory->createNamed('name', 'money', 1234.56, array(
+            'currency' => false,
+        ));
+
+        $this->assertWidgetMatchesXpath($form->createView(), array('id' => 'my&id', 'attr' => array('class' => 'my&class')),
+'/input
+    [@id="my&id"]
+    [@type="text"]
+    [@name="name"]
+    [@class="my&class form-control"]
+    [@value="1234.56"]
```

Comment:
does this prevent regression also?

Reply:
What kind of regression do you mean?

Reply:
if a `input-group-addon` is rendered again, this xpath would still match no?

Reply:
Indeed, the test still passes if any extra element is rendered, as long as the `<input>` is at first nesting level. But this actually applies to most of these tests with XPath. This could (should?) probably be improved but in a separate PR IMO. WDYT?

Reply:
Yeah, but now we just test `form_widget_simple` again; which is probably already covered in some other test. For this new test i'd expect it to cover the change; no rendering of input addons in case of currency=false.

Reply:
Added more predicates.

Reply:
Nice!